### ISSUE TITLE
Infinite scrolling behaviour in right panel

### ISSUE THREAD

**Issue by daniellekirkwood** [vector-im/element-meta#934]
### Your use case

Threads work revealed a design problem with the right panel (align the threads list to the top...)

We have deprioritised this work from Threads as it's non-trivial however, the intended design should be implemented at some point so I'm moving discussion links here so that we can track them moving forwards.

--- 
**From @janogarcia** [Here](https://github.com/vector-im/element-web/issues/21502#issuecomment-1138637261)

So, the only issue pending to be addressed is the infinite scrolling behavior. More specifically, the limitation of the current implementation ([BACAT](https://github.com/matrix-org/matrix-react-sdk/blob/develop/docs/scrolling.md?rgh-link-date=2022-05-26T14%3A26%3A26Z#bacat-bottom-aligned-clipped-at-top-scrolling)), that prevents us from ordering the scrollable events from top to bottom.

I’ve been thinking more about this lately and would like to share some updates:

I’m dropping the requirement (I already checked it off) for the top-to-bottom alignment in the thread timeline (Align timeline to the top of the timeline container.), which is, by far, the less critical one. I’d even say that it can work almost equally well either with BACAT or TACAB scrolling.
I still consider the case for the thread list to be relevant, though (Align timeline to the top of the timeline container.). We (the design team) would still want users to have scrolling lists where the items are ordered from top to bottom. That doesn’t only make sense for the Thread List but other contexts as well, like the Mentions/Notifications panel or the Files panel.
From a UX perspective, we can’t presume that BACAT scrolling will work and serve best every possible scenario and interaction need (e.g., we already use the top-to-bottom scrolling pattern in the room list on the left sidebar and the people list on the right sidebar, although not for infinite items).

We consider top-to-bottom scrolling a core interaction pattern worth the effort of solving. It surely has a considerable implementation cost, but it also has a similar or higher product/UX cost not having it.

Now, given that we want to reuse this behavior in different contexts (mentions panel, files panel), meaning it’s not a Threads-specific limitation but a core interaction issue affecting our product in general, I think we could move this requirement out of the scope of the Exit Beta milestone for Threads and handle it separately.

So, to recap:

We are dropping the requirement for the vertical alignment of the thread timeline altogether.
We’d still want to order the thread list from most recently updated on the top to oldest on the bottom. We would handle it separately from the Threads project, treating it as a global app enhancement instead.
What do you think?

### Are you willing to provide a PR?

No

**Comment by daniellekirkwood** [comment]
Moving to web repo as it's not an issue on mobile.
